great_expectations/cli/datasource.py

```diff
@@ -672,6 +672,7 @@ def create_expectation_suite(
     click.prompt(msg_prompt_what_will_profiler_do, default="Enter", hide_input=True)
 
     cli_message("\nProfiling {0:s}...".format(generator_asset))
+    run_id = datetime.datetime.now().isoformat().replace(":", "") + "Z"
```

Comment:
Update to the newer default run_id (we should confirm this is done everywhere)

```suggestion
    run_id = datetime.datetime.utcnow().strftime("%Y%m%dT%H%M%S.%fZ")
```